src/PrestaShopBundle/Translation/Provider/ThemeProvider.php

```diff
@@ -88,4 +130,37 @@ public function setThemeName($themeName)
 
         return $this;
     }
+
+    /**
+     * @param null $theme
+     * @return \Symfony\Component\Translation\MessageCatalogue
+     */
+    public function getDatabaseCatalogue($theme = null)
```

Comment:
same as before ... but very weird (you dont use ``$theme``), you should rely on Objects instead of strings